qutebrowser/utils/usertypes.py

```diff
@@ -25,7 +25,7 @@
 
 import operator
 import collections.abc
-import enum as pyenum
+from enum import Enum, IntEnum
```

Comment:
Please do `import enum` here, and use `enum.Enum` etc. here - I don't like importing types from modules, as it makes it difficult to keep track what's coming from where.